src/common/simple-manifest.js

```diff
@@ -0,0 +1,31 @@
+const getSimpleManifest = function (manifest) {
+  // This function generates a simplified manifest with the public and private
+  // variable fields removed. All public and private variables are defined in the
+  // environment field of each container
+
+  const simpleManifest = { ...manifest['manifest'] }
```

Comment:
This code might be more clear/readable if we explicitly chose the fields we need out of the manifest rather than taking all of them and selectively deleting/overwriting.

Reply:
Makes sense